### DISCUSSION TITLE
Supporting timezones in the Wordpress plugin

### DISCUSSION THREAD

**Discussion by jeznag** [code4recovery/12-step-meeting-list#1157] [Q&A]
Hi,
I'm experimenting with the TSML wordpress plugin for Debtors Anonymous Australasia: https://debtorsanonymous.asia/meetings/

We've got meetings in multiple timezones and I can't figure out how to specify the timezone for a particular meeting. 

e.g. https://debtorsanonymous.asia/meetings/?meeting=carlisle-perth-da should be in Perth time but it shows the meeting in Sydney time (which is the default timezone for my wordpress instance)

Is there documentation for this?

**Comment by joshreisner** [comment]
hi @jeznag sorry, the short answer is no, you currently can't specify time zone on the TSML meeting entry screen -- we have an [open issue](https://github.com/code4recovery/12-step-meeting-list/issues/930) to address this.

the plugin was originally designed for - and mainly serves - smaller regions within a single time zone. there are a few tz-spanning service entities ([example](http://alcoholics-anonymous.eu/meetings/)) that do use it to list meetings, they must use the "Legacy UI" because that isn't timezone-aware like the newer "TSML UI" is. so that's one alternative: switch to Legacy UI and leave it to users to interpret the timezones.

there is another possible alternative, which is to enter your meetings in a google sheet with a timezone column, TSML UI can either connect to this directly directly or through a proxy we developed. more info on using TSML UI without the plugin is here: https://tsml-ui.code4recovery.org/
